package.json

```
@@ -2127,6 +2127,248 @@
 											}
 										]
 									},
+									{
+										"key": "languageId:coq",
+										"name": "coq",
+										"type": "bindings",
+										"bindings": [
+											{
+												"key": ".",
+												"name": "Proof goto current point",
+												"icon": "debug-start",
+												"type": "command",
+												"command": "extension.coq.interpretToPoint"
+											},
+											{
+												"key": "b",
+												"name": "Proof step back",
+												"icon": "debug-step-back",
+												"type": "command",
+												"command": "extension.coq.stepBackward"
+											},
+											{
+												"key": "f",
+												"name": "Proof step forward",
+												"icon": "debug-step-over",
+												"type": "command",
+												"command": "extension.coq.stepForward"
+											},
+											{
+												"key": "g",
+												"name": "Go to the current focus location",
+												"icon": "sync",
+												"type": "command",
+												"command": "extension.coq.moveCursorToFocus"
+											},
+											{
+												"key": "o",
+												"name": "Open proof view",
+												"icon": "open-preview",
+												"type": "command",
+												"command": "extension.coq.proofView.open"
+											},
+											{
+												"key": "v",
+												"name": "View the proof-state at the cursor position",
+												"icon": "eye",
+												"type": "command",
+												"command": "extension.coq.proofView.viewStateAt"
+											},
+											{
+												"key": "G",
+												"name": "Proof goto end",
+												"icon": "debug-continue",
+												"type": "command",
+												"command": "extension.coq.interpretToEnd"
+											},
+											{
+												"key": "a",
+												"name": "Ask prover",
+												"icon": "question",
+												"type": "bindings",
+												"bindings": [
+													{
+														"key": "a",
+														"name": "About",
+														"icon": "info",
+														"type": "command",
+														"command": "extension.coq.query.prompt.about"
+													},
+													{
+														"key": "c",
+														"name": "Check",
+														"icon": "check",
+														"type": "command",
+														"command": "extension.coq.query.prompt.check"
+													},
+													{
+														"key": "l",
+														"name": "Locate",
+														"icon": "location",
+														"type": "command",
+														"command": "extension.coq.query.prompt.locate"
+													},
+													{
+														"key": "p",
```

```suggestion
														"key": "a",
```

The [coq layer in spacemacs](https://develop.spacemacs.org/layers/+lang/coq/README.html#prover-queries) have print in `a` key. Not sure it does the same thing or if we should follow it.

---

It does the same thing. But I think `Print` should be `p` instead of `a`. In fact, I don't know why coq layer use `a`.

---

That makes sense. Let's use `p`.